cli/src/test/java/com/devonfw/tools/ide/tool/vscode/VscodeTest.java

```diff
@@ -0,0 +1,38 @@
+package com.devonfw.tools.ide.tool.vscode;
+
+import org.junit.jupiter.api.Test;
+
+import com.devonfw.tools.ide.context.AbstractIdeContextTest;
+import com.devonfw.tools.ide.context.IdeTestContext;
+
+public class VscodeTest extends AbstractIdeContextTest {
+
+  private static final String PROJECT_VSCODE = "vscode";
+
+
+  @Test
+  public void testVscode() {
+
+    // arrange
+    IdeTestContext context = newContext(PROJECT_VSCODE);
+    Vscode vscodeCommandlet = new Vscode(context);
+
+    // install
+    vscodeCommandlet.install();
+
+    // run
+    vscodeCommandlet.run();
```

Comment:
I would assume the `run` call is not necessary for the installation to complete successfully, please remove